demux/demux_playlist.c

```diff
@@ -207,7 +207,7 @@ static int parse_m3u(struct pl_parser *p)
             char probe[PROBE_SIZE];
             int len = stream_read_peek(p->real_stream, probe, sizeof(probe));
             bstr data = {probe, len};
-            if (ext && data.len > 10 && maybe_text(data)) {
+            if (ext && data.len && maybe_text(data)) {
```

Comment:
we *could* also change it to `data.len >=2`
that would cover a file with a 1 byte filename plus a line terminator

not sure if it matters however

Reply:
Yeah that works. We can go with that.